mod/01_rating/set/ltype_rtype/metric/wikirate_company.rb

```diff
@@ -41,7 +45,7 @@ def default_menu_args args
     # TODO: change the css so that we don't need the extra logo class here
     #   and we can use a logo_link view on the type/company set
     text = content_tag :div, class: "logo" do
-             field_subformat(:image)._render_core size: 'small'
+             card.company_card.format.field_nest :image, view: :core, size: 'small'
```

Comment:
(this was the reason I asked about the rubocop integration. In textmate, it's usually obvious when you create a new style issue.)